frontend/src/component/project/ProjectList/ProjectGroup.tsx

```diff
@@ -23,12 +24,25 @@ const StyledCardLink = styled(Link)(({ theme }) => ({
     pointer: 'cursor',
 }));
 
-export const ProjectGroup: React.FC<{
+type ProjectGroupProps<T extends { id: string } = IProjectCard> = {
```

Comment:
is the extends because we're spreading remaining props?

Reply:
It's because we are using `projects[id]` for both mapped `ProjectCardComponent` and links.